src/Compilers/CSharp/Portable/CodeGen/EmitExpression.cs

```diff
@@ -2645,6 +2652,12 @@ private void EmitSizeOfExpression(BoundSizeOfOperator boundSizeOfOperator)
             EmitSymbolToken(type, boundSizeOfOperator.SourceType.Syntax);
         }
 
+        private void EmitMethodTokenExpression(BoundMethodToken node)
+        {
+            _builder.EmitOpCode(ILOpCode.LoadMethodTokenAsInteger_i4);
```

Comment:
I didn't/don't like the idea much either, but couldn't think of a workable alternative. I'd love to learn a better way.